scripts/docs-deploy/utils.ts

```diff
@@ -0,0 +1,50 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import {$} from 'zx';
+import {installDepsForDocsSite, InstallOptions} from './docs-deps-install';
+
+/** Absolute path to the `angular/components` project root. */
+export const projectDir = path.join(__dirname, '../..');
+
+/** Interface describing a site target for the docs-app. */
+export class SiteTarget {
+  constructor(public firebaseSiteId: string, public remoteUrl: string) {}
+}
+
+/** Object capturing all site targets for the docs-app. */
+export const sites = {
+  stable: new SiteTarget('ng-comp-test', 'https://material.angular.io'),
```

Comment:
Are these names correct for the site id?

Reply:
Yeah, they are not 😄  this was just for testing. I will wire up my custom project for testing. @andrewseguin proposed using a test project temporarily to ensure it works as expected.